posts/2023-10-20-open-liberty-with-cargo-tracker-ee10.adoc

```diff
@@ -0,0 +1,119 @@
+
+After you start VSCode IDE, open the Cargo Tracker project by selecting **Explorer** > **Open Folder**. Select the `cargotracker` directory.
+
+image::/img/blog/cargo-tracker-vscode-open-folder.png[Open Cargo Tracker project,width=70%,align="center"]
+
+The `cargo-tracker` project appears under the `LIBERTY DASHBOARD` extension in the Explorer sidebar. To run Cargo Tracker, right-click the `cargo-tracker` project and select the **Start...** option.
+
+image::/img/blog/cargo-tracker-vscode-start.png[Start Cargo Tracker,width=70%,align="center"]
+
+You can then run dev mode with the Open Liberty profile active by typing in `-Popenliberty` in the pop-up entry field and then press the `enter/return` key. 
+
+image::/img/blog/cargo-tracker-vscode-parameters.png[Start Cargo Tracker with dash P parameter,width=60%,align="center"]
+
+After starting with the custom parameter for the Open Liberty profile, a terminal window in the IDE opens and the Liberty Maven Plugin starts the application. Liberty is running in dev mode. You can safely ignore the shrinkwrap features warning, message-driven bean warning, and the AggregateObjectMapping nested foreign key warning, as they don't affect the application functionality.
+
+image::/img/blog/cargo-tracker-vscode-running.png[Cargo Tracker running in dev mode,width=70%,align="center"]
+
+
+== Run unit tests
+
+To ensure the Cargo Tracker application started successfully, you can run the tests by pressing the `enter/return` key from the command-line session where you started dev mode. If the tests pass, you see a similar output to the following example:
+
+image::/img/blog/cargo-tracker-ee10-unit-test-result.png[Cargo Tracker unit tests result,width=70%,align="center"]
+
+
+== Access the application
+
+Once the application is running, you can access it through a browser of your choice at http://localhost:8080/cargo-tracker/ and start experimenting with it. You can track routed cargo through the Public Tracking interface, book and route cargo through the Administration interface, and update cargo object events using the Event Logger. You can stop the Liberty at any time by pressing `CTRL-C` from the command-line session where you started dev mode. 
+
+image::/img/blog/cargo-tracker-ee10-dashboard-event-logger.png[Cargo Tracker Dashboard and Event Logger,width=60%,align="center"]
+
+For more information about of each of the interfaces, can check out the https://github.com/eclipse-ee4j/cargotracker/tree/master#exploring-the-application[Exploring the Application] section of the project README. 
+
+
+== Nice work! Where to next?
+
+You've just set up Cargo Tracker and experienced Open Liberty with an enterprise-level application on your own machine! To learn the Cargo Tracker deeply, you can update the source code or use the debugger to step through the code.
+
+Check out the https://openliberty.io/guides/[Open Liberty guides] for more information on using Jakarta EE and MicroProfile with Open Liberty. 
```

Comment:
"for more information and interactive tutorials to set you through using more Jakarta EE and MicroProfile APIs with Open Liberty". Nice to include the fact that they are also interactive and more tutorial like rather than just documentation.